catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrFilterDelegate.java

```diff
@@ -290,6 +290,11 @@ public SolrQuery propertyIsLike(String propertyName, String pattern, boolean isC
     }
 
     String searchPhrase = escapeSpecialCharacters(pattern);
+
```

Comment:
❓ Shouldn't there be impacted or new test cases for this change?

Reply:
Looks like the anyText unit tests have been commented out for years `DDF-314: COmmented out until the ANY_TEXT functionality is added back in - then these tests can be activated`

Reply:
There are many unit and integration tests doing this type of query that did not regress.  The query to Solr is returning the same results but it can now do it faster when there is a lot of data indexed.  We do not have a good way to do the kind of performance test necessary to verify this change currently.

Reply:
I was referring to unit testing ;-)

Reply:
Test added.